pkg/tsdbctl/delete.go

```diff
@@ -87,11 +92,45 @@ func (ic *delCommandeer) delete() error {
 			return err
 		}
 	}
-	err = ic.rootCommandeer.adapter.DeleteDB(ic.delConfig, ic.force, from, to)
+
+	if !ic.force {
+		confirmedByUser, err := getConfirmation(
+			fmt.Sprintf("You are about to delete the '%s' table. Are you sure?", ic.rootCommandeer.v3iocfg.Path))
```

Comment:
add what you expect the user to enter "Are you sure? y/n"

Reply:
I can confirm that when running the command, ` [y/n]` appears at the end of the question.